google/monitoring/v3/monitoring_gapic.yaml

```diff
@@ -0,0 +1,331 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.monitoring.v3
+  python:
+    package_name: google.monitoring.v3
+  go:
+    package_name: google.golang.org/google/monitoring/v3
+  csharp:
+    package_name: Google.Monitoring.V3
+  ruby:
+    package_name: Google::Monitoring::V3
+  php:
+    package_name: Google\Monitoring\V3
+interfaces:
+- name: google.monitoring.v3.GroupService
+  collections:
+  - name_pattern: projects/{project}
+    entity_name: project
+  - name_pattern: projects/{project}/groups/{group}
+    entity_name: group
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 60000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 60000
+    total_timeout_millis: 600000
```

Comment:
That is what config gen gave me. In case it is not intended I created an issue: https://github.com/googleapis/toolkit/issues/263